### PR TITLE
Also add an HTTP/3 frame.

### PR DESCRIPTION
**Pull request by davidben**
@bencebeky

### CODE REVIEW

File: draft-davidben-http-client-hint-reliability.md [hunk 1]
```diff
@@ -270,13 +323,22 @@ and other unmatched entries are ignored.
...
-* The server's Accept-CH preferences may change while existing connections are open. Those connections will have outdated ACCEPT_CH frames. While the server could send a new one, the frame may not arrive in time for the next request. Moreover, if the HTTP serving frontend is an intermediary like a CDN, it may not be proactively notified of origin server changes.
+* The server's Accept-CH preferences may change while existing connections are
+  open. Those connections will have outdated ACCEPT_CH frames. While the server
+  could send a new frame, it may not arrive in time for the next request.
+  Moreover, if the HTTP serving frontend is an intermediary like a CDN, it may
+  not be proactively notified of origin server changes.
 
-* HTTP/2 allows connection reuse across multiple origins (Section 9.1.1 of {{RFC7540}}). Some origins may not be listed in the ACCEPT_CH frame, particularly if the server used a wildcard X.509 certificate.
+* HTTP/2 and HTTP/3 allow connection reuse across multiple origins (Section
+  9.1.1 of {{RFC7540}} nad Section 3.4 of {{I-D.ietf-quic-http}}). Some origins
```
Comment (bencebeky) [inline draft-davidben-http-client-hint-reliability.md]: s/nad/and/

File: draft-davidben-http-client-hint-reliability.md [hunk 2]
```diff
@@ -270,13 +323,22 @@ and other unmatched entries are ignored.
...
+  may not be listed in the ACCEPT_CH frame, particularly if the server used a
+ wildcard X.509 certificate.
```
Comment (bencebeky) [inline draft-davidben-http-client-hint-reliability.md]: add one more space of indentation

**Review by bencebeky — APPROVED** [review]

File: draft-davidben-http-client-hint-reliability.md (hunk 1 above)
Comment (davidben) [reply draft-davidben-http-client-hint-reliability.md]: Fixed.

File: draft-davidben-http-client-hint-reliability.md (hunk 2 above)
Comment (davidben) [reply draft-davidben-http-client-hint-reliability.md]: Fixed.

**Pull request merged**